azure/template.json

```diff
@@ -60,6 +60,9 @@
         },
         "sharedEnvResourceGroup": {
             "type": "string"
+        },
+        "findApprenticeshipTrainingUrl": {
```

Comment:
Hey Dan, why is the App settings in two files? I believe this should only be in the config file.